code/modules/projectiles/projectile.dm

```diff
@@ -161,6 +161,8 @@
 	var/turf/target_turf = get_turf(A)
 
 	if(!prehit(A))
+		if(forcedodge)
+			loc = target_turf

```

Comment:
I don't know if that does anything bad in relation to projectile code and I don't want to break anything finding out